pkg/webhook/node/plugins/cpunormalization/node_cpu_normalization.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cpunormalization
+
+import (
+	"context"
+	"fmt"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+	"github.com/koordinator-sh/koordinator/pkg/util"
+	"github.com/koordinator-sh/koordinator/pkg/webhook/node/plugins"
+)
+
+var (
+	CPUNormalizationResourceList = []corev1.ResourceName{
+		corev1.ResourceCPU,
```

Comment:
I think we shall only mutate resource CPU here because the SLO controller would use the amplified resource CPU to calculate amplified batch and mid CPU which does not need another amplification.

Reply:
Good idea, logically it should be like this